backend/typescript/middlewares/validators/util.ts

```diff
@@ -79,7 +79,7 @@ export const validateUserEmail = (userEmail: string): boolean => {
   );
 };
 
-export const validateFormResponses = (
+export const validateMap = (
   formResponses: any,
```

Comment:
nit: should we rename this variable to map as well if we're making this a general validator? @amanda-guo 
```suggestion
  map: any,
```